keyboards/atom47/keymaps/default/keymap.c

```diff
@@ -13,25 +13,25 @@
 #define _______ KC_TRNS
 
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
-[_MA] = LAYOUT_ansi(
+[_MA] = LAYOUT(
```

Comment:
@mechmerlin did we want these to be `LAYOUT` or `LAYOUT_ansi`?

Reply:
This is a drop-in replacement for the vortex core, which does not come in ISO variants. 

Based on https://atomkb.eu/atom47/ this board also does not support ISO variants, so there really is no need to say that it is LAYOUT_ansi.

Reply:
Okay, just wanted to make sure. :)